src/components/MapView/Analyser/index.tsx

```diff
@@ -98,12 +103,15 @@ function Analyser({ classes }: AnalyserProps) {
           (LayerDefinitions[hazardLayerId] as WMSLayerProps)?.serverLayerName
         ]
       : null;
+    // if (Number(new URLSearchParams(history.location.search).get('date'))) {
```

Comment:
Remove this comment? Also, did you test if the `date` url query works after removing this? We'll need a solution that both allows this `useEffect` to work alongside the url query code. If you need help coming up with a way to do this let me know :+1:

Reply:
Removed!